src/app/pipes/token.pipe.ts

```diff
@@ -1,30 +1,34 @@
 import { Pipe, PipeTransform } from '@angular/core';
 import { UserToken } from '../models/usertoken';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
 
 @Pipe({
     name: 'token',
 })
 export class TokenPipe implements PipeTransform {
```

Comment:
Just out of curiocity, this seems something that would better suited as a component instead, any specific reason you decided to go with the pipe way?

Reply:
I just used the Pipe because we had it before... But this should really be component. Thanks!